spec/integration/common.js

```diff
@@ -59,16 +63,27 @@ Given('editor is open', () => {
   cy.wait(2000);
 });
 
+Given('add workflow popup is open', () => {
+  click('Add Workflow Button');
+});
+
 When('I click on {string}', click);
 When('I select {string} from {string}', select);
 When('I type {string} in {string}', type);
-When('I confirm on {string} popup', popupConfirm);
+When('I confirm on {string}', popupConfirm);
+When('I cancel on {string}', popupCancel);
 Then('I should see {string} in {string}', assertInputValueEQ);
 Then('I should not see {string} in {string}', assertNotInputValueNotEQ);
 
 Then('{string} should {string}', (element, expectation) => {
   const cExpectation = expectation.replace(/\s/g, '.');
   $(element).should(cExpectation);
 });
+Then('{string} should contain {string} {string}', (element, expectation, childElement) => {
+  $(element).find(selector(childElement)).should("have.length", expectation);
+});
+Then('{string} should have {string} {string} style', (element, cssValue, cssProperty) => {
+  $(element).should("have.css", cssProperty, styleValueSelector(cssValue));
+});
```

Comment:
these rules belongs to `workflows` test case. Can you move it to `workflows/steps.js`?